docs/source/internals/sandbox.rst

```diff
@@ -51,7 +51,12 @@ play around with Oscar.
     While installing Oscar is straightforward, some of Oscar's dependencies
     don't support Windows and are tricky to be properly installed, and therefore
     you might encounter some errors that prevent a successful installation.
-    
+
+In order to compile uWSGI, dependency of sandbox, you'll need to install
```

Comment:
@sasha0 may I correct the English grammar here, to:

`In order to compile uWSGI, which is a dependency of the sandbox, you will first need to install the Python development headers with:`

Reply:
Of course you may. That's why the review was requested 😉

Reply:
Ok done.